.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1 +1 @@
-*Make sure that you generate site HTML with `bundle exec jekyll build`, and include the changes to the HTML in your pull request also. See README.md for more information. Please remove this message.*
+<!-- *Make sure that you generate site HTML with `bundle exec jekyll build`, and explain the changes to the HTML in your pull request. See README.md for more information.* -->
```

Comment:
I think 'include' was intended here. Rest seems OK.

Reply:
Personally I think "include" is less clear than "explain", when in the same sentence as the bundle command, since it sounds like it's talking about the command.

Reply:
It's talking about the output from Jekyll. Markdown and HTML changes both have to be included in the PR.

Reply:
Oh, all right. I'll replace explain with include.